lib/wickedbase.pm

```diff
@@ -442,16 +442,17 @@ sub validate_macvtap {
     my ($self, $macvtap_log) = @_;
     my $ref_ip     = $self->get_ip(type => 'host',    netmask => 0, is_wicked_ref => 1);
     my $ip_address = $self->get_ip(type => 'macvtap', netmask => 0);
-    my $cmd_text   = "./check_macvtap $ref_ip $ip_address > $macvtap_log 2>&1 &";
-    sleep(30);    # OVS on a worker is slow sometimes to change and we haven't found better way how to handle it
+    my $cmd_text   = "./check_macvtap $ref_ip $ip_address > $macvtap_log 2>&1 & export MYPID=$!";
     type_string($cmd_text);
```

Comment:
I think we can now also just call `(assert_)script_run()` like in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/8242/files#diff-7c6669e0bb0ed1f0eed4a62dc81f200dR580 or?

Reply:
let's try it